gui/packages/desktop/src/main/index.js

```diff
@@ -335,8 +402,13 @@ const ApplicationMain = {
       }
     });
 
-    ipcMain.on('daemon-connection-status', (event) => {
-      event.sender.send('daemon-connection-status-reply', this._connectedToDaemon);
+    ipcMain.on('get-state', (event) => {
```

Comment:
FYI used in renderer process to obtain the daemon connection status, current tunnel state and settings.